package.json

```diff
@@ -145,6 +148,7 @@
 		"html-entities": "^2.3.2",
 		"jose": "^1.28.1",
 		"jsonwebtoken": "8.5.1",
+		"jszip": "^3.10.1",
```

Comment:
Looks like you added two libs to create zip files and used only one.
It would be nice to have some reasoning for the lib you picked, because in a way you are setting the project standard for creating zip files with this.

Reply:
Done.